.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+    "presets": [
+        "@babel/preset-env"
```

Comment:
This is currently required for jest but I expect soon we can drop babel for tests.